packages/aws-cdk-lib/cx-api/lib/features.ts

```diff
@@ -824,6 +825,17 @@ export const FLAGS: Record<string, FlagInfo> = {
     recommendedValue: true,
   },
 
+  //////////////////////////////////////////////////////////////////////
+  [EFS_MOUNTTARGET_ORDERINSENSITIVE_LOGICAL_ID]: {
+    type: FlagType.BugFix,
+    summary: 'Enable this feature flag to manage mount targets of the elastic file system more align with its subnet',
+    detailsMd: `
+      If this flag is specified, Subnets of the EFS will be modifed correctly. 
+      If it is not set, exiting EFSs won't be affected.`,
+    introducedIn: { v2: 'V2·NEXT' },
```

Comment:
```suggestion
    introducedIn: { v2: 'V2NEXT' },
```

I've been guilty of the same